### ISSUE TITLE
A brief review of various packages

### ISSUE THREAD

**Issue by Sisyphuss** [JuliaPlot/juliaplot_docs#5]
I start with giving a brief review of various packages. The work is located at: https://docs.google.com/document/d/1EU1V12auPgwGkPuLJ7rpCXAeF7jmTGtcI1uXQjzbZio/edit?usp=sharing

You can give any comment, advice or feedback in this issue.

**Comment by lobingera** [comment]
I'd rather expect this to be a local document in markdown.

**Comment by Sisyphuss** [comment]
It seems to me that "julia-plot" is written in Julia. Why Github tells me that it is written in Objective-J?

**Comment by lobingera** [comment]
julia-plot seems to have only 1 source file and this is named plot.j (not .jl), so ...

**Comment by Sisyphuss** [comment]
But the syntax of plot.j is in Julia!

**Comment by mbauman** [comment]
(Julia-plot is so old it dates from when .j was the extension for julia)

**Comment by tbreloff** [comment]
This is hilarious. I didn't even know the extension used to be ".j". Ok we can get rid of julia-plot.

**Comment by wildart** [comment]
@Sisyphuss Please, make a PR to this repo of your document for sharing purposes, so further discussion could continue. BTW, markdown format is appreciated.

**Comment by wildart** [comment]
I really like this [Producing Simple Graphs with R](http://www.harding.edu/fmccown/r/) intro. It would be nice to have similar functionality description for every plotting package.

**Comment by waldyrious** [comment]
Just quickly stepping in to comment that I don't think we should remove any packages from the list, just mark them as deprecated, as has been suggested [elsewhere](https://github.com/JuliaPlot/juliaplot_docs/issues/1#issuecomment-134317395).

I will be rather busy for the next few days, but will collaborate in this documentation as I mentioned [earlier](https://github.com/JuliaPlot/juliaplot_docs/issues/1#issuecomment-134610330), as soon as possible. In that regard, I support @wildart's suggestion to move the google doc to a md page either checked into this repo, or in the [wiki](https://github.com/JuliaPlot/juliaplot_docs/wiki).

**Comment by Sisyphuss** [comment]
My work isn't complete yet. Is it a bit too early to do this? I'm still thinking about the difference of these packages and what is more appropriate way to present it and use it. But if everyone requires me to do this, I'm ok to  make a PR. 

However, concerning the "Producing Simple Graphs with R", it should *not* be something we should struggle on. As mentioned in #1, the tutorial should be the job of the author of each separated package.

**Comment by tbreloff** [comment]
> However, concerning the "Producing Simple Graphs with R", it should not be something we should struggle on. As mentioned in #1, the tutorial should be the job of the author of each separated package.
> 
Yes, but the best thing we could do is to make a template for this sort of tutorial, so that package authors have an easy time just filling in their own code for how to make the plots (without figuring out design or content, which is probably the hardest part). I support using the R tutorial for inspiration, and if someone can produce a template with blank code sections and invalid links to screenshots of the plots, we could churn out the tutorials quickly and in consistent style.

**Comment by Sisyphuss** [comment]
> Just quickly stepping in to comment that I don't think we should remove any packages from the list, just mark them as deprecated, as has been suggested elsewhere.

Just a quick note: it's not always trivial to know whether a package is deprecated. For example, GoogleChart.jl has no activities since January. Is it because that the package works always well or it is abandoned?

**Comment by tbreloff** [comment]
I started the wiki... how do people feel about putting everything there and just linking to it from the README?

**Comment by waldyrious** [comment]
Just as a heads-up, I started the comparison table. It's in a [Google Sheets file](https://docs.google.com/spreadsheets/d/12yKNculcdruxdPTVMVRLRf1dXO5AGc2Np8yVPv3VVlI/edit#gid=0) for now, to make it easy to make large edits, but as soon as it's mostly stable and filled out, I'll move it to the [wiki](https://github.com/JuliaPlot/juliaplot_docs/wiki) where we can edit the table as markdown.

I'm yet to integrate info from @cormullion's [wiki page](https://en.wikibooks.org/wiki/Introducing_Julia/Plotting), and @Sisyphuss' [google doc](https://docs.google.com/document/d/1EU1V12auPgwGkPuLJ7rpCXAeF7jmTGtcI1uXQjzbZio/edit#) (which will hopefully soon be [moved to our wiki](https://github.com/JuliaPlot/juliaplot_docs/pull/7)). Any help filling out the table (or suggestions about edits to make, columns to add or remove, etc.) is most welcome. You should be able to make edit suggestions directly in the document -- let me know if it works properly.

After this step is done, and if someone doesn't do it before me, I'll get to the examples documentation, i.e. a systematic listing of input (code) and output (screenshots) for common plots across the various packages.

**Comment by waldyrious** [comment]
> I started the wiki... how do people feel about putting everything there and just linking to it from the README?

Agreed, that would make contributing easier.
